features/delete_reference.feature

```diff
@@ -0,0 +1,11 @@
+#language: fr
+
+Fonctionnalité: Supprimer les références existantes
+
+  Scénario: Supprimer la référence à droite
+
+    Soit "Les fées : Conte" le document principal
+    Et une session active avec mon compte
+    Quand je vais supprimer les références existantes
```

Comment:
You should have a given name:
- in the preconditions,
- in the action,
- in the postconditions.

See the edition scenarios for examples.